app/assets/stylesheets/components/_language_selector.scss

```diff
@@ -0,0 +1,16 @@
+.language-selector__select {
+  margin-bottom: $baseline-unit*2;
+}
+
+.language-selector__delete {
+  margin-left: 0.5em;
+  cursor: pointer;
```

Comment:
Should these really be link tags? The designs make them look like links, but maybe they shouldn't actually be links? Since they don't technically hyperlink anywhere. Or is that OK?

Advice appreciated.